### DISCUSSION TITLE
When setting a team through require-reviewers plugin can we have it where we can set the required approvals for that team (or per team)

### DISCUSSION THREAD

**Discussion by MilesNorton** [linear-b/gitstream#480] [Request an Automation]
At the moment I have a two workflows that both set the required-reviewers to 2 separate teams since we want both teams to approve it.
I would also like to require 2 approvals from team A and 1 approval from team B. At the moment this doesn't work since set-required-approvals doesn't link to the teams that are required to approve.

**Comment by vim-zz** [comment]
Hi, can you share your CM automation here?

**Reply by clemyp** [reply]
We also have this issue and would like this functionality.

We would like 1 person from each team to check the PR, currently this check can be bypassed if 2 members from 1 team approve.


  ## example setup
  team_review_required:
    if:
      - {{ files | match(list=code_paths) | some }}
    run:
      - action: add-reviewers@v1
        args:
          reviewers: [team-platform-engineering, team-triumph]
      - action: set-required-approvals@v1
        args:
          approvals: 2
      - action: add-comment@v1
        args:
          comment: |
            This PR affects the Azure Firewall rules and requires review from the both teams.
